projects/vdk-core/src/vdk/internal/builtin_plugins/config/log_config.py

```diff
@@ -57,12 +68,36 @@ def configure_loggers(
         "stream": "ext://sys.stderr",
     }
 
+    syslog_url, syslog_port, syslog_sock_type, syslog_enabled = syslog_args
+
+    if syslog_sock_type not in SYSLOG_SOCK_TYPE_VALUES_DICT.keys():
+        errors.log_and_throw(
+            to_be_fixed_by=errors.ResolvableBy.CONFIG_ERROR,
+            log=log,
+            what_happened=f"Provided configuration variable for {SYSLOG_SOCK_TYPE} has invalid value.",
+            why_it_happened=f"VDK was run with {SYSLOG_SOCK_TYPE}={syslog_sock_type}, however {syslog_sock_type} is invalid value for this variable.",
+            consequences=errors.MSG_CONSEQUENCE_DELEGATING_TO_CALLER__LIKELY_EXECUTION_FAILURE,
+            countermeasures=f"Provide either valid value for {SYSLOG_SOCK_TYPE} or install database plugin that supports this type. "
+            f"Currently possible values are {SYSLOG_SOCK_TYPE_VALUES_DICT.keys()}",
+        )
+
+    _SYSLOG_HANDLER = {
+        "level": "DEBUG",
+        "class": "logging.handlers.SysLogHandler",
+        "formatter": "detailedFormatter",
+        "address": (syslog_url, syslog_port),
+        "socktype": SYSLOG_SOCK_TYPE_VALUES_DICT[syslog_sock_type],
+        "facility": "user",
+    }
+
+    handlers = ["consoleHandler", "SysLog"] if syslog_enabled else ["consoleHandler"]
```

Comment:
```suggestion
    handlers = ["consoleHandler"] 
    if syslog_enabled:
        handlers += ["SysLog"]
```

Seems a bit cleaner to me. But it's a nit, feel free to ignore it.

Reply:
noted